docs/scripts_md/ImagingUpload.md

```diff
@@ -0,0 +1,163 @@
+# NAME
+
+NeuroDB::ImagingUpload -- Provides an interface to the uploaded imaging file
+
+# SYNOPSIS
+
+    use NeuroDB::ImagingUpload;
+
+    my $imaging_upload = &NeuroDB::ImagingUpload->new(
+                           \$dbh,
+                           $TmpDir_decompressed_folder,
+                           $upload_id,
+                           $patient_name,
+                           $profile,
+                           $verbose
+                         );
+
+    my $is_candinfovalid = $imaging_upload->IsCandidateInfoValid();
+
+    my $output = $imaging_upload->runDicomTar();
+    $imaging_upload->updateMRIUploadTable('Inserting', 0) if ( !$output );
+
+
+    my $output = $imaging_upload->runTarchiveLoader();
+    $imaging_upload->updateMRIUploadTable('Inserting', 0) if ( !$output);
+
+    my $isCleaned = $imaging_upload->CleanUpDataIncomingDir($uploaded_file);
+
+# DESCRIPTION
+
+This library regroups utilities for manipulation of the uploaded imaging file
+and updates of the `mri_upload` table according to the upload status.
+
+## Methods
+
+### new($dbhr, $uploaded\_temp\_folder, $upload\_id, ...) (constructor)
+
+Create a new instance of this class. This constructor needs the location of
+the uploaded file. Once the uploaded file has been validated, it will be
+moved to a final destination directory.
+
+INPUT:
```

Comment:
did we say we will have INPUTS when we have multiple? (i.e. INPUT with the `s`)

Reply:
I don't believe there is any convention yet. I can add the s when there are many inputs. Another thing to re-verify once everything is merged. ;)